apps/dashboard/src/modules/user/components/UserSettingsComponent.vue

```diff
@@ -80,6 +87,58 @@ import { useConfirm } from "primevue/useconfirm";
 import { useToast } from "primevue/usetoast";
 import apiService from "@/services/ApiService";
 import { handleError } from "@/utils/errorUtils";
+import { useUserStore } from "@sudosos/sudosos-frontend-common";
+
+const status: Ref<string> = ref("");
+
+async function startScan() {
+  try {
+    if (!('NDEFReader' in window)) {
+      status.value = "Web NFC is not supported on this device.";
+      console.log('no');
+      toast.add({
+            severity: "error",
+            summary: "Unsupported",
+            detail: "Browser is not mobile and does not have Web NFC.",
+            life: 3000,
+          });
+    } else {
```

Comment:
This can be unnested into an if statement with a return

```ts
if (!('NDEFReader' in window)) {

```

Reply:
fixed